fftw-src doesn't compile

Probably the problem is with `fs_extra 1.2`:

```
error[E0063]: missing field `content_only` in initializer of `fs_extra::dir::CopyOptions`
  --> fftw-src/build.rs:52:10
   |
52 |         &fs_extra::dir::CopyOptions {
   |          ^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `content_only`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0063`.
error: could not compile `fftw-src`.
```

--- a/fftw-src/Cargo.toml
+++ b/fftw-src/Cargo.toml
@@ -15,6 +15,6 @@ links = "fftw3"
 [build-dependencies]
 anyhow = "1.0.31"
 cc = "1.0"
-fs_extra = "1.1"
+fs_extra = "~1.2.0"
 ftp = "3.0"
 zip = "0.5"

--- a/fftw-src/build.rs
+++ b/fftw-src/build.rs
@@ -55,6 +55,7 @@ fn build_unix(out_dir: &Path) {
             buffer_size: 64000,
             copy_inside: true,
             depth: 0,
+            content_only: false,
         },
     )
     .unwrap();

